codelab/01_kpt-validate/tutorial.md

```diff
@@ -0,0 +1,259 @@
+# Validate Your Configurations in skaffold Workflow
+
+<walkthrough-disable-features toc></walkthrough-disable-features>
+
+## Introduction
+
+#### What is kpt?
+
+Kpt is [an OSS tool](https://github.com/GoogleContainerTools/kpt) for Kubernetes packaging, which uses a standard format to bundle, publish, customize, update, and apply configuration manifests.
+
+#### What kpt can help you
+    
+*  You will get an hands-on off-the-shelf experience about the **GitOps** CI/CD workflow in skaffold.
+*  You can validate each of your config changes **declaratively**.
+*  You **won't** encounter **version conflict** if the config hydration (a.k.a kustomize) mismatch with the deployment tool (e.g. kubectl). 
+*  You can prune your resources accurately with [a three-way merge strategy](https://kubectl.docs.kubernetes.io/pages/app_management/field_merge_semantics.html). 
+
+#### What you'll learn
+    
+*  How to add a validation to your config changes. 
+*  How to define validation rules in the form of a declarative configuration.
+*  How to use the validation in kustomized resources. 
+
```

Comment:
The content here isn't consistent with the bullet points in `README.md`. Is this intentional?

Reply:
Yeah, I'm planning to merge the validate-1 and deploy-2 README.md to a single file. But since they are currently two separate PRs. I'd prefer not making the change. You can ignore README.md since it would be changed.

Reply:
works for me 👍🏼